v3/lints/rfc/lint_name_constraint_not_fqdn_test.go

```diff
@@ -0,0 +1,58 @@
+package rfc
+
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"testing"
+
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/test"
+)
+
+func TestBeginsWithPeridFQDN(t *testing.T) {
```

Comment:
I won't consider it a blocker for merging but if you're up to it I'd prefer to see these unit test functions implemented as one test function that uses [a table-driven design](https://github.com/golang/go/wiki/TableDrivenTests).